### PR TITLE
Rename geo types.

### PR DESCRIPTION
**Pull request by BillWhite**
This is the community request.

The geo types need to be renamed.  The changes are:
Java:
  PointType -> GeographyPointValue
  VoltPointType -> VoltGeographyPointType

C++:
  PointType -> GeographyPointValue

https://issues.voltdb.com/browse/ENG-9212

### CODE REVIEW

File: src/ee/common/GeographyPoint.hpp [hunk 1]
```diff
@@ -15,8 +15,8 @@
  * along with VoltDB.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef EE_COMMON_POINT_HPP
-#define EE_COMMON_POINT_HPP
+#ifndef EE_COMMON_GEOGRAPHY_POINT_HPP
```
Comment (wolffcm) [inline src/ee/common/GeographyPoint.hpp]: I didn't know that git/GitHub could handle a file rename and changes to a file in the same commit, and make it look nice in the diff.  This is nice!  No other SCM I've used has dealt with this very well.

File: src/ee/common/GeographyPoint.hpp [hunk 2]
```diff
@@ -31,20 +31,20 @@ namespace voltdb {
 /**
  * A class for representing instances of geo-spatial points.
  */
-class Point {
+class GeographyPointValue {
```
Comment (wolffcm) [inline src/ee/common/GeographyPoint.hpp]: Hmm.  This C++ class is called GeographyPointValue (like Java), but the C++ class for geography is still just Geography.  Maybe change Geography to GeographyValue?

**Comment by wolffcm** [comment]
I just had the one question about the C++ Geography class.  Otherwise, looks good.

File: src/ee/common/GeographyPoint.hpp (hunk 2 above)
Comment (BillWhite) [inline src/ee/common/GeographyPoint.hpp]: That seems right.  I'll change it.

**Pull request closed without merging**